contributors/devel/testing.md

```diff
@@ -225,3 +225,40 @@ version and the watch cache test is skipped.
 ## End-to-End tests
 
 Please refer to [End-to-End Testing in Kubernetes](e2e-tests.md).
+
+## Running your contribution in the Kubernetes CI
+Once you open a PR, prow will run pre-submit tests in CI.
```

Comment:
possibly mention `/ok-to-test` and `/retest` and/or link to some of the PR / test related bot commands?

Reply:
Added, thanks!